### PR TITLE
dependency(react-native-paper): setup + test components on SettingsView page

### PR DESCRIPTION
**Pull request by Calinteodor**

### CODE REVIEW

File: react/features/settings/components/native/SettingsView.js [hunk 1]
```diff
@@ -15,13 +16,25 @@ import {
...
-import FormSectionHeader from './FormSectionHeader';
+import FormSectionAccordion from './FormSectionAccordion';
+import styles, { THUMB_COLOR } from './styles';
 
 /**
  * Application information module.
  */
 const { AppInfo } = NativeModules;
 
+const theme = {
```
Comment (saghul) [inline react/features/settings/components/native/SettingsView.js:27]: Let's wait until Vlad's PR lands and put this in base/ui.

File: react/features/settings/components/native/SettingsView.js [hunk 2]
```diff
@@ -149,73 +164,108 @@ class SettingsView extends AbstractSettingsView<Props, State> {
...
-                        label = 'settingsView.version'>
-                        <Text>
-                            {`${AppInfo.version} build ${AppInfo.buildNumber}`}
-                        </Text>
-                    </FormRow>
-                    <FormSectionHeader
-                        label = 'settingsView.advanced' />
-                    {this._renderAdvancedSettings()}
-                </ScrollView>
+                <PaperProvider theme = { theme }>
```
Comment (saghul) [inline react/features/settings/components/native/SettingsView.js:167]: Let's wait until Vlad's PR lands and then add some JitsiThemeProvider in base/ui like the web is doing.

File: webpack.config.js [hunk 3]
```diff
@@ -174,6 +174,7 @@ const config = {
     ].filter(Boolean),
     resolve: {
         alias: {
+            'react-native$': require.resolve('react-native-web'),
```
Comment (saghul) [inline webpack.config.js:177]: Do we need this in the end?

File: package.json [hunk 4]
```diff
@@ -77,13 +78,15 @@
...
+    "react-native-paper": "^4.8.1",
     "react-native-sound": "github:jitsi/react-native-sound#3fe5480fce935e888d5089d94a191c7c7e3aa190",
     "react-native-splash-screen": "3.2.0",
     "react-native-svg": "12.1.0",
     "react-native-svg-transformer": "0.14.3",
     "react-native-url-polyfill": "1.2.0",
     "react-native-watch-connectivity": "0.4.3",
-    "react-native-webrtc": "github:react-native-webrtc/react-native-webrtc#d58dbb5599e16f7bdc918a9f384e36946187a5d6",
+    "react-native-web": "^0.15.7",
+    "react-native-webrtc": "github:react-native-webrtc/react-native-webrtc#1343580c0322f265ff0fb22722ac5501c5fd77ad",
```
[the remark this answers is not in the document]
Comment (Calinteodor) [reply package.json:89]: Sorry about this. Got confused on rebase.

File: webpack.config.js (hunk 3 above)
Comment (Calinteodor) [reply webpack.config.js]: Removed react-native-web .

File: package-lock.json [hunk 5]
```diff
@@ -4,38 +4,36 @@
   "lockfileVersion": 1,
```
[the remark this answers is not in the document]
Comment (Calinteodor) [reply package-lock.json:4]: npm -v 6.14.10

**Pull request closed without merging**